packages/module/plugin-module-vue/README.md

```diff
@@ -0,0 +1,60 @@
+# @modern-js/plugin-module-vue
+
+The Vue plugin of Modern.js Module.
+
+The Vue plugin provides support for building Vue 3 components. The plugin internally integrates [esbuild-plugin-vue3](https://github.com/pipe01/esbuild-plugin-vue3) and [@vue/babel-plugin-jsx](https://github.com/vuejs/babel-plugin-jsx).
+
+**Now, it doesn't support that write jsx/tsx in sfc, look forward your contributing!**
+
+## Usage
+
+```ts modern.config.ts
+import { moduleTools, defineConfig } from '@modern-js/module-tools';
+import { modulePluginVue } from '@modern-js/plugin-module-vue';
+export default defineConfig({
+  plugins: [
+    moduleTools(),
+    modulePluginVue(),
+  ],
+});
+```
+
+## Options
+
+### vueJsxPluginOptions
```

Comment:
We do not need to maintain a detailed document in the README.md, just add a link to the plugin document is enough.